### ISSUE TITLE
Cannot reset pw from remote pc

### ISSUE THREAD

**Issue by yongalee** [anvil-works/anvil-runtime#8]
What I did
1. set sendgrid SMTP relay (from their website)
2. created a file smtp.yaml containing:

smtp-host: smtp.sendgrid.net
smtp-port: 465
smtp-user: apikey
smtp-encryption: ssl
smtp-password:

3. launched MyApp (has email/passwd auth) using 
anvil-app-server --config-file smtp.yaml --app MyApp

4. everything works fine upto this point

5. did pw reset using forgot my password link in the login form

6. MyApp sent me an email, and  the link in the email looks something like this:
You have requested a password reset for your account: myid@gmail.com

Click here to reset your password:
https://u14813360.ct.sendgrid.net/ls/click?upn=1S426CWQBw8xnOCf1Pp4CeV6IUdfuf9N7T38aE0EdaPWzbQJXQfTTl9OP4EnDi7TkAadCFJe5Mna2RNvPvt0u9dA-2BWmxnoMvQjM8VM11Fo3hqx25r1M0XdTMryduC5ZAUIh-_XHBkSK-2Fx8jqI-2FRPnHM6x0oi9HmGGzawtEshi3sFZywswKMa2NcUPb2kWsRoTTPAdi91HRqdtMrMmbTJVG6Y156TYDyVpoBfAFQ8nUDyCRX-2BS3-2BXSzeC90gCaencWs6g4D6pCOAqsNjT5aoAtCwB-2FFbQnTYH3LniSFl4vRYAZK-2B6vHik2RNxzxpLjfmrCThQGdn3ZeqQhzNTNWWs8b44ZE4Kr3jw6-2FIJ8six9j-2BFRXdI-3D

# The Problem:
when I click above link from a remote pc, the link starts with
 localhost:3030/_/email-pw-reset/blahblahblah --> which obviously does not work cause i'm remote.
I changed the localhost part to the external ip of my local server manually, and then it worked.

So, what I'm wondering is, if there is a way that I can tweak a little bit so that my pw reset link points to the external ip address as a default link. 

So, in a nutshell,
instead this pw reset link (  localhost:3030/_/email-pw-reset/blahblahblah ),
how can I send to user a pw reset link with  externalIP:3030/_/email-pw-reset/blahblahblah

**Comment by meredydd** [comment]
Hi there,

This isn't a bug -- it looks like you've got the wrong `origin` set in your configuration. See the README for how to set your origin correctly.
